assume/world.py

```diff
@@ -593,7 +607,7 @@ async def run_simulation(self):
 
         start_ts = calendar.timegm(self.start.utctimetuple())
         end_ts = calendar.timegm(self.end.utctimetuple())
-        pbar = tqdm(total=end_ts - start_ts)
+        pbar = tqdm(total=end_ts - start_ts, leave=True)
```

Comment:
https://tqdm.github.io/docs/tqdm/
> leave: bool, optional
> If [default: True], keeps all traces of the progressbar upon termination of iteration. If None, will leave only if position is 0.

So this does not add anything?

Reply:
oh yeah true, in my build in docs it showed default value as false.